google/cloud/bigtable/error.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Class for error status."""
+
+
+class Status:
+    """A status, comprising a code and a message.
+
+    See: `Cloud APIs Errors <https://cloud.google.com/apis/design/errors>`_
+
+    This is a thin wrapper for ``google.rpc.status_pb2.Status``.
```

Comment:
is it possible to use the pb2 Status as is? Would be nice to have less handwritten code to look after :)

Reply:
I was doing it in the first draft, then decided for consistency I should probably wrap it. I think I saw in the Java reference implementation it was wrapped and figured I'd better follow suit. I think my personal preference would be to return the bare pb2, if that's not considered a violation of the contract for user exposed APIs.

Reply:
It also just occurred to me, that the wrapper class might be preferable for documentation purposes, as it causes the interface to get documented in the Sphinx docs. Otherwise, the user has to track down the protocol buffer definitions. 

I'll defer to whatever call you want to make here.

Reply:
Sorry for the delay here - following up on this. I'll update here when I can :)

Reply:
Oh no, I'm sorry for the delay in getting back to you. I had fumbled something in my email filters and didn't see you had responded to the initial draft, so I thought I was still waiting.

Reply:
I think we're going to keep this - resolving